src/Reader/XLSX/SheetMergeCellsReader.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenSpout\Reader\XLSX;
+
+use OpenSpout\Common\Exception\IOException;
+use OpenSpout\Reader\Common\XMLProcessor;
+use OpenSpout\Reader\Wrapper\XMLReader;
+
+final class SheetMergeCellsReader
+{
+    public const XML_NODE_MERGE_CELL = 'mergeCell';
+    public const XML_ATTRIBUTE_REF = 'ref';
+
+    /** @var string Path of the XLSX file being read */
+    private readonly string $filePath;
+
+    /** @var string Path of the sheet data XML file as in [Content_Types].xml */
+    private readonly string $sheetDataXMLFilePath;
+
+    /** @var XMLReader The XMLReader object that will help read sheet's XML data */
+    private readonly XMLReader $xmlReader;
+
+    /** @var XMLProcessor Helper Object to process XML nodes */
+    private readonly XMLProcessor $xmlProcessor;
+
+    /** @var string[] Merged cells list */
+    private array $mergeCells = [];
+
+    /**
+     * @param string       $filePath             Path of the XLSX file being read
+     * @param string       $sheetDataXMLFilePath Path of the sheet data XML file as in [Content_Types].xml
+     * @param XMLReader    $xmlReader            XML Reader
+     * @param XMLProcessor $xmlProcessor         Helper to process XML files
+     */
+    public function __construct(
+        string $filePath,
+        string $sheetDataXMLFilePath,
+        XMLReader $xmlReader,
+        XMLProcessor $xmlProcessor
+    ) {
+        $this->filePath = $filePath;
+        $this->sheetDataXMLFilePath = $this->normalizeSheetDataXMLFilePath($sheetDataXMLFilePath);
+        $this->xmlReader = $xmlReader;
```

Comment:
What's the point of requesting and saving the `$xmlReader` if it's already injected in the `$xmlProcessor`?

Reply:
@Slamdunk I guess for `$this->xmlReader->openFileInZip(...)`